### PR TITLE
Add basic error handler.

### PR DESCRIPTION
**Pull request by seanohue**
Addresses #1

### CODE REVIEW

**Comment by seanohue** [comment]
I accidentally pushed some commits that were in a branch....
Let me try again.

**Comment by seanohue** [comment]
Fixed the custom code woopsie

**Comment by seanohue** [comment]
@shawncplus  Any feedback on this?

**Comment by shawncplus** [comment]
Could you add a comment about why if there's an error number you don't throw an exception?

**Comment by stennie** [comment]
This would be a super handy PR to have merged, particularly for use with Neuro. Without this error handler, quitting the Neuro app also causes the Ranvier server to shutdown.
